include/ck/tensor_operation/gpu/device/impl/device_pool3d_fwd_impl.hpp

```diff
@@ -29,8 +30,9 @@ template <typename InDataType,
           ck::index_t KThreadClusterSize,
           ck::index_t MThreadSliceSize,
           ck::index_t KThreadSliceSize,
-          ck::index_t InSrcOutDstVectorSize>
-struct DevicePool3dFwd_Input_N_Di_Hi_Wi_C_Output_N_Do_Ho_Wo_C
+          ck::index_t InSrcOutDstVectorSize,
+          bool IsFastestDimReduced>
```

Comment:
Can you add comments to give details on how the user should set `IsFastestDimReduced` ?  I think it is hard to use this when `embedding transform` is used (eg. implicit gemm convolution)

Reply:
As discussed in teams, replace IsFastestDimReduced with layout parameter